content/en/apps/guides/privacy/privacy.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Privacy & Data Protection Policy"
+linkTitle: "Privacy"
+weight: 
+description: >
+  This is the current version of Medic Mobile’s Privacy and Data Protection Policy, which applies to the Community Health Toolkit.
+keywords: 
+relatedContent: >
+  apps/reference/app-settings/patient_reports
+---
+
+{{% alert title="Note" %}}
+This policy is reviewed and updated periodically by our Responsible Data Working Group and is intended as a resource for the CHT community. 
+{{% /alert %}}
+
+If you have any questions, please reach out to our Data Protection Officer at support@medicmobile.org. 
+
+## Table of Contents
+- [Our responsible data promise](#our-responsible-data-promise)
+- [Compliance with applicable data regulations and policies](#compliance-with-applicable-data-regulations-and-policies)
+- [Data and protected health information](#data-and-protected-health-information)
+  - [Use of software to collect and process data](#use-of-software-to-collect-and-process-data)
+  - [Protected health information policy](#protected-health-information-policy)
+- [Security practices](#security-practices)
+  - [Technology security overview](#technology-security-overview)
+  - [Security training for our Partners](#security-training-for-our-partners)
+- [Inquiries with Medic Mobile’s Data Protection Officer](#inquiries-with-medic-mobiles-data-protection-officer)
+
+
+
+## Our responsible data promise
+
+As a non-profit organization, Medic Mobile’s mission is to advance good health and human flourishing by building open source technology with and for hard-to-reach communities. We value humanity, creativity, initiative, solidarity, and openness. Being responsible stewards of people’s data is critically important to our mission. 
+
+We use the concept of Responsible Data (RD) to outline our collective duty to prioritise and respond to the ethical, legal, social, and privacy-related challenges that come from using data in new and different ways in healthcare, advocacy, and social change. We have an obligation to account for unintended consequences of working with data by:
+
+1. Prioritising people’s rights to consent, privacy, security and ownership when using data in social change and advocacy efforts,
```

Comment:
This bullet point ends in a comma, and the second with a period. Is that what's intended?

I'm actually not sure that either of these two points require punctuation at the end because they're not complete sentences.